packages/core/src/components/StarRating.tsx

```diff
@@ -54,6 +73,16 @@ const styles = StyleSheet.create({
     flexDirection: "row",
     alignItems: "center",
   },
+  touchContainer: {
+    display: "flex",
+    flexDirection: "row",
+    position: "absolute",
+    top: 0,
+    right: 0,
+    left: 0,
+    bottom: 0,
+    zIndex: 111,
```

Comment:
Is it possible to make this a more reasonable value like `1` or `2`?

Reply:
Fixed